### PR TITLE
Issue #SB-23271 fix: adding modifications in df apis

### PR DESCRIPTION
**Pull request by VenkatKadiveti**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @VenkatKadiveti,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://724.qa.docs.sunbird.org/3.7.0/ .<br/> **Note:** Only _3.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://724.qa.docs.sunbird.org/3.7.0/apis/discussionForum// <br /> - http://724.qa.docs.sunbird.org/3.7.0/apis/discussionForum// <br />  <br/> **Modified sections are -** <br />

File: apis/discussionForum/discussionForum.yaml [hunk 1]
```diff
@@ -0,0 +1,5451 @@
...
+      - User can create multiple categories, under each category he can perform multiple discussions.
+      - User can post the questions.
+      - User can reply to the post.
+      - User can upvote or downvote to the posts.
+  version: 1.0.0
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+servers:
+- url: https://staging.sunbirded.org/discussions/api
```
Comment (SMYALTAMASH) [inline apis/discussionForum/discussionForum.yaml:15]: IT should be DNS/api/discussion

**Comment by keshavprasadms** [comment]
Hello @VenkatKadiveti,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://724.qa.docs.sunbird.org/3.7.0/ .<br/> **Note:** Only _3.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://724.qa.docs.sunbird.org/3.7.0/apis/discussionForum// <br /> - http://724.qa.docs.sunbird.org/3.7.0/apis/discussionForum// <br />  <br/> **Modified sections are -** <br />

File: apis/discussionForum/discussionForum.yaml [hunk 2]
```diff
@@ -0,0 +1,5451 @@
...
+info:
+  title: Discussions Forum
+  description: |-
+    The discussions forum allows the user to perticipate in multiple discussion.
+      - User can create multiple categories, under each category he can perform multiple discussions.
+      - User can post the questions.
+      - User can reply to the post.
+      - User can upvote or downvote to the posts.
+  version: 1.0.0
+externalDocs:
```
Comment (Sunitashelar) [inline apis/discussionForum/discussionForum.yaml:11]: Are lines  11 - 13 required?

File: apis/discussionForum/discussionForum.yaml [hunk 3]
```diff
@@ -0,0 +1,5451 @@
...
+              schema:
+                $ref: '#/components/schemas/discussionForumCategoryDetails'
+        "400":
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/discussionForumError'
+      x-swagger-router-controller: Categories
+  /discussion/category/{cid}:
```
Comment (Sunitashelar) [inline apis/discussionForum/discussionForum.yaml:154]: lines 154 - 160 are the same as lines 107 - 113. is this correct?

**Comment by keshavprasadms** [comment]
Hello @VenkatKadiveti,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://724.qa.docs.sunbird.org/3.7.0/ .<br/> **Note:** Only _3.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://724.qa.docs.sunbird.org/3.7.0/apis/discussionForum// <br /> - http://724.qa.docs.sunbird.org/3.7.0/apis/discussionForum// <br />  <br/> **Modified sections are -** <br />

File: apis/discussionForum/discussionForum.yaml [hunk 4]
```diff
@@ -0,0 +1,5451 @@
...
+                $ref: '#/components/schemas/discussionForumError'
+      x-codegen-request-body-name: body
+      x-swagger-router-controller: Categories    
+  /discussion/v2/categories/{cid}/state:
+    put:
+      tags:
+      - categories
+      summary: Enable category
+      description: |-
+        This app allow the user to delete a topic in nodebb
```
Comment (Sunitashelar) [inline apis/discussionForum/discussionForum.yaml]: Is this correct? does not match title of API

File: apis/discussionForum/discussionForum.yaml [hunk 5]
```diff
@@ -0,0 +1,5451 @@
...
+              schema:
+                $ref: '#/components/schemas/discussionForumError'
+      x-codegen-request-body-name: body
+      x-swagger-router-controller: Categories  
+    delete:
+      tags:
+      - categories
+      summary: DisableCategories
+      description: |-
+        This app allow the user to delete a topic in nodebb
```
Comment (Sunitashelar) [inline apis/discussionForum/discussionForum.yaml]: Is this correct? does not match title

File: apis/discussionForum/discussionForum.yaml [hunk 6]
```diff
@@ -0,0 +1,5451 @@
...
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/discussionForumError'
+      x-codegen-request-body-name: body
+      x-swagger-router-controller: Categories  
+    delete:
+      tags:
+      - categories
+      summary: DisableCategories
```
Comment (Sunitashelar) [inline apis/discussionForum/discussionForum.yaml:476]: This is a repeat of line 387. please check

File: apis/discussionForum/discussionForum.yaml [hunk 7]
```diff
@@ -0,0 +1,5451 @@
...
+              schema:
+                $ref: '#/components/schemas/discussionForumError'
+      x-swagger-router-controller: Users
+  /discussion/user/v1/create:
+    post:
+      tags:
+      - users
+      summary: Create new user in nodebb OIDC
+      description: |-
+        This api is used for creating the user in Nodebb.
```
Comment (Sunitashelar) [inline apis/discussionForum/discussionForum.yaml]: This description is the same as on line 679. clarify the difference in the two APIs

File: apis/discussionForum/discussionForum.yaml [hunk 8]
```diff
@@ -0,0 +1,5451 @@
...
+    delete:
+      tags:
+      - topics
+      summary: Follow a Post
+      description: |-
+        This app allow the user to add tags to a topic in nodebb
+          - The endpoint for **Category** is `/v2/topics/{tid}/tags`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+          - Backend Route: http://nodebb-service:4567/discussions/api/v2/topics/{tid}/tags
+      operationId: DaleteTagsToTopic
```
Comment (Sunitashelar) [inline apis/discussionForum/discussionForum.yaml:1542]: This does not match the title. please check.

File: apis/discussionForum/discussionForum.yaml [hunk 9]
```diff
@@ -0,0 +1,5451 @@
...
+      x-swagger-router-controller: Topics  
+    delete:
+      tags:
+      - topics
+      summary: Follow a Post
+      description: |-
+        This app allow the user to add tags to a topic in nodebb
+          - The endpoint for **Category** is `/v2/topics/{tid}/pin`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+          - Backend Route: http://nodebb-service:4567/discussions/api/v2/topics/{tid}/pin
```
Comment (Sunitashelar) [inline apis/discussionForum/discussionForum.yaml:1627]: please check the backend route

File: apis/discussionForum/discussionForum.yaml [hunk 10]
```diff
@@ -0,0 +1,5451 @@
...
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/discussionForumError'
+      x-swagger-router-controller: Groups
+  /discussion/groups/{slug}:
+    get:
+      tags:
+      - groups
+      summary: Get group list
```
Comment (Sunitashelar) [inline apis/discussionForum/discussionForum.yaml:1845]: is this a repeat of line 1812?

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

**Comment by keshavprasadms** [comment]
Hello @VenkatKadiveti,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://724.qa.docs.sunbird.org/3.7.0/ .<br/> **Note:** Only _3.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://724.qa.docs.sunbird.org/3.7.0/apis/discussionForum// <br /> - http://724.qa.docs.sunbird.org/3.7.0/apis/discussionForum// <br />  <br/> **Modified sections are -** <br />

**Comment by Sunitashelar** [comment]
Closing the PR without merging due to conflicts. Spoke to Venkat and new PR will be sent.

**Pull request closed without merging**